bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -132,6 +132,13 @@
 
 	<!-- Channel Type - Generic Things -->
 
+	<channel-type id="windowPosition">
+		<item-type>Rollershutter</item-type>
+		<label>@text/channel-type.velux.windowPosition.label</label>
+		<description>@text/channel-type.velux.windowPosition.description</description>
+		<category>Window</category>
+	</channel-type>
```

Comment:
You are not setting a min/max state like for the "position" type ?

Reply:
Actually the position type does NOT have min/max. Whereas the vanePosition type DOES. But really I think that all three of them should have min/max. => WDYT?

Reply:
My feeling is that it would be better.

Reply:
Ok. Added.

Reply:
PS to be consistent I added it now to all position channels